manage_deps/workflows.rst

```diff
@@ -39,16 +39,16 @@ So the layout will be:
       main.cpp
 
 
-Now you are ready for building:
+Now you are ready to build:
 
 .. code-block:: bash
    
       $ cmake ../example-hello -G ...
       $ cmake --build . --config Release
       
-With this configuration you are typically working in just one configuration of the project, 
-that will be destroyed and create a new build with a new configuration with different settings
-if needed.
+In effect, with this setup, we have created a separate configuration of the project, without
+affecting its default one. The benefit is that we can experiment freely, and even erase it and
```

Comment:
By the way, for future pull requests, is it better if I make them on the "develop" branch? I've just noticed it now.